### DISCUSSION TITLE
Ready For Batch

### DISCUSSION THREAD

**Discussion by sammish** [lavaloon-eg/ksa_compliance#192] [Q&A] [answered]
Hello Team,

What does "Ready for Batch" mean, and how does it sync? Kindly clarify.

![image](https://github.com/user-attachments/assets/3ae6a289-49b2-4b4c-956f-ad3e1647978f)

@mhaggag

**Comment by mhaggag** [comment, marked as answer]
Hello,

I converted the issue to a discussion. I monitor both, so please use discussions if it's not an actual issue/bug.

In `ZATCA Business Settings`, you can set the integration mode to `Live` or `Batches`. In live mode, invoices are immediately sent to ZATCA when submitted. This is meant for B2B scenarios and low-workload scenarios (invoices per second). In batch mode, invoices are sent to ZATCA every two hours. `Ready For Batch` means the invoice is ready to be sent but has not yet been sent.

Note that in live mode, the `Sales Invoice Additional Fields` starts in `Ready For Batch` for a few seconds after submitting the invoice during which it's being submitted to ZATCA, after which it should get updated to `Accepted`, `Rejected`, etc. depending on the ZATCA response.

The reference documentation on the wiki is [here](https://lavaloon.com/wiki/zatca-business-settings) and [here](https://lavaloon.com/wiki/process-flow#batch-process)
